### PR TITLE
Update connect-to-microsoft-365-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
moved the sdk line to to a note at the top of the article.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8299

File: microsoft-365/enterprise/connect-to-microsoft-365-powershell.md [hunk 1]
```diff
@@ -38,6 +38,11 @@ Currently, the Azure Active Directory PowerShell for Graph module doesn't comple
 
 ## What do you need to know before you begin?
 
+>[!Note]
+> The AzureAD module is being replaced by Powershell graph SDK.
+> Get started with the Microsoft Graph PowerShell SDK
+> You can use the Microsoft Graph PowerShell SDK to access all Microsoft Graph APIs.For more information, see [Get started with the Microsoft Graph PowerShell SDK](/powershell/microsoftgraph/get-started?view=graph-powershell-beta)
+
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/connect-to-microsoft-365-powershell.md]:
```suggestion
>[!NOTE]
> The Azure Active Directory Module is being replaced by the Microsoft Graph PowerShell SDK. You can use the Microsoft Graph PowerShell SDK to access all Microsoft Graph APIs. For more information, see [Get started with the Microsoft Graph PowerShell SDK](/powershell/microsoftgraph/get-started).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**